js/richObjectStringParser.js

```
@@ -45,8 +45,12 @@
 
 			_.each(matches, function(parameter) {
 				parameter = parameter.substring(1, parameter.length - 1);
-				var parsed = self.parseParameter(parameters[parameter]);
+				if (!parameters.hasOwnProperty(parameter) || !parameters[parameter]) {
+					// Malformed translation?
```

should this be reported instead of failing silently?

---

Well only thing would be  console.log, but that is disabled for non-debug anyway, so I don't think it's important.

---

so, wouldn't it be helpful for debugging? for the app dev to to find out where an issues occurs?

---

whatever I was thinking, yes of course, amended